### DISCUSSION TITLE
Basic shell for RP6502

### DISCUSSION THREAD

**Discussion by jthwho** [picocomputer/community#91] [Q&A]
I've started work on a basic shell that can call a list of func(int argc, char *argv[]) style functions.  It started as a quality of life thing for me as I'm working on some custom VGA stuff, but I figured others might find it useful.  Probably a good place for some basic file management stuff once we've got dirent functions.

You can find it here: https://github.com/jthwho/rp6502-shell

Jason

**Comment by WojciechGw** [comment]
thank You Jason :)
please take a look at
https://github.com/WojciechGw/cc65-rp6502os

**Reply by jthwho** [reply]
Sorry about the delay, I didn't see this message.  Very cool project!
